beacon_node/network/src/sync/network_context.rs

```diff
@@ -223,9 +224,17 @@ impl<T: BeaconChainTypes> SyncNetworkContext<T> {
             network_send,
             execution_engine_state: EngineState::Online, // always assume `Online` at the start
             request_id: 1,
-            blocks_by_root_requests: <_>::default(),
-            blobs_by_root_requests: <_>::default(),
-            data_columns_by_root_requests: <_>::default(),
+            // true = enfore max_requests as returned for blocks_by_root. We always request a single
```

Comment:
I'm confused - when do we not want to enforce max requests returned? it looks like this is always set to true. Is there a follow up PR where this could be false?

Reply:
Yeah, sorry should have added a note. For the all the `*_by_range` requests